sql/informix-sql/InformixSQLLexer.g4

```diff
@@ -257,5 +273,8 @@ SPACES: [ \u000B\t\r\n] -> channel(HIDDEN);
 
 UNEXPECTED_CHAR: .;
 
+fragment NEWLINE_EOF    : NEWLINE | EOF;
```

Comment:
What's the meaning of introducing such token? I don't see usings.

Reply:
It's useful in the future.